book/src/sidecar-containers.md

```diff
@@ -26,3 +26,34 @@ The Kubernetes development team maintains the following Kubernetes CSI Sidecar C
 * [node-driver-registrar](node-driver-registrar.md)
 * [cluster-driver-registrar](cluster-driver-registrar.md)
 * [livenessprobe](livenessprobe.md)
+
+
+## Versioning
```

Comment:
The content here looks good. Maybe add subheadings so we can link to them directly

```
## Versioning

Every sidecar has a minimum, maximum and recommended Kubernetes version.

### Minimum Version
...

### Maximum Version
...

### Recommended Version
...
```

And then in the headers for the table can link directly to these subheadings here for those who are wondering what they mean